composer.lock

```diff
@@ -219,16 +219,16 @@
         },
         {
             "name": "guzzlehttp/guzzle",
-            "version": "6.2.1",
+            "version": "6.2.2",
```

Comment:
It do not seems a major problem as these are patch versions and we are supposed to do them. However right way of including a package is to use command `composer require vendor/package`

By using that command, only required package gets updated. Since all packages in this diff are updated (even though patch versions), we need to to test it completely before going live.

I'll do quick testing but it will take a bit of time as I'm on diwali vacations and having limited internet access in my home town through my mobile.

In general, it seems good and should not have any problem. Thanks a lot for PR. I'll merge it soon.